### ISSUE TITLE
Add --report option to CLI

### ISSUE THREAD

**Issue by RhetTbull** [RhetTbull/osxphotos#253]
When exporting in the CLI it's not easy to know exactly what was exported (original, edited, live, burst, RAW, etc) without combing through the verbose output.  A `--report FILE.CSV` option would export a report in CSV format, of every action taken.

e.g.
```csv
uuid,original_filename,path,path_exported,exported,skipped,updated,touched,exiftool,islive,isburst,raw,etc
4D521201-92AC-43E5-8F7C-59BC41C37A96,IMG_0001.JPG,/Users/rhet/Pictures/Test-10.15.7.photoslibrary/originals/4/4D521201-92AC-43E5-8F7C-59BC41C37A96.jpeg,/Users/rhet/Desktop/export/2020-11/IMG_0001.JPG,True,False,False,True,True,False,False,False,...
```

**Comment by RhetTbull** [comment]
See also issue #252

**Comment by RhetTbull** [comment]
Closed in release v0.37.1
